image.css

```diff
@@ -0,0 +1,49 @@
+img {
+  object-fit: cover;
+}
+
+.img-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2px;
+}
+
+@media only screen and (max-width: 600px) {
+  .img-responsive {
+    height: 8rem;
+    width: 8rem;
+  }
+}
+@media only screen and (min-width: 600px) {
+  .img-responsive {
+    height: 12rem;
+    width: 12rem;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .img-responsive {
+    height: 17rem;
+    width: 17rem;
+  }
+}
+
+@media only screen and (min-width: 992px) {
```

Comment:
You can use this breakpoints for media queries. You can find it in the figma link.
![image](https://user-images.githubusercontent.com/16834982/152779945-2c6aa606-2d13-4227-ae4e-e4f00c656254.png)

Reply:
Done